dev-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest==2.9.2
```

Comment:
The test dependencies are actually defined in the `setup.py` file: https://github.com/mobify/python-panopticon/blob/master/setup.py#L8. Seems like the installation process isn't documented well enough in the README to explain that. I'll update that. But the TL;DR is running `pip install -e ".[test]"` will install all the test dependencies as well.